spec/acceptance/ferm_spec.rb

```diff
@@ -32,7 +32,7 @@ class { 'ferm':
     manage_configfile => true,
     manage_initfile   => #{manage_initfile}, # CentOS-6 does not provide init script
     forward_policy    => 'DROP',
-    output_policy     => 'DROP',
+    output_policy     => 'ACCEPT',
```

Comment:
I think the changes in this file aren't related to the PR?

Reply:
hum ... it was in the PR #81 . Surprised i am. :D